docs/swiperefresh.md

```diff
@@ -0,0 +1,30 @@
+# Swipe Refresh layout for Jetpack Compose
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.accompanist/accompanist-swiperefresh)](https://search.maven.org/search?q=g:com.google.accompanist)
+
+TODO
+
+## Usage
+
+``` kotlin
+// TODO
```

Comment:
We should add this in, right?

Reply:
Yep! I was going to take a pass through docs at the end.

Reply:
Will follow up on this in #324